gdrive_sync/api.py

```diff
@@ -167,7 +167,7 @@ def get_confirm_token(response: requests.Response) -> str:
     return response
 
 
-def stream_to_s3(drive_file: DriveFile):
+def stream_to_s3(drive_file: DriveFile, prefix=settings.DRIVE_S3_UPLOAD_PREFIX):
```

Comment:
I think `settings.DRIVE_S3_UPLOAD_PREFIX` is evaluated during file import rather than during the function call here. Can we change this to `prefix=None` and then have a `if prefix is None: prefix = settings.DRIVE_S3_UPLOAD_PREFIX` at the beginning of the function?

Reply:
Thanks, changed it